static/sass/_instance_list.scss

```diff
@@ -0,0 +1,42 @@
+.p-instance-table {
+  @media screen and (max-width: 1400px) {
+    th,
+    td {
+      &:nth-child(4),
+      &:nth-child(5) {
+        display: none;
+      }
+    }
+  }
+}
+
+.p-instance-table-with-panel {
+  th,
+  td {
+    &:nth-child(4),
+    &:nth-child(5) {
+      display: none;
+    }
+  }
+}
+
+.u-row:hover,
+.u-row-selected {
+  background-color: $colors--light-theme--background-hover;
```

Comment:
I think the hover and selected should slightly differ. Lets follow up with design on this question.

Reply:
I checked this and the design for now seems to use the same.